File: 백준/Bronze/30802. 웰컴 키트/웰컴 키트.py
import sys

N = int(sys.stdin.readline())
sizes = list(map(int, sys.stdin.readline().split()))
T, P = map(int, sys.stdin.readline().split())

t_sum = sum([(size + T - 1) // T for size in sizes])
p_bundle, p_single = divmod(N, P)

print(t_sum)
print(p_bundle, p_single)

File: 백준/Bronze/30802. 웰컴 키트/웰컴 키트.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

        int N = Integer.parseInt(br.readLine());
        String[] sizesStr = br.readLine().split(" ");
        String[] bundlesStr = br.readLine().split(" ");
        int T = Integer.parseInt(bundlesStr[0]);
        int P = Integer.parseInt(bundlesStr[1]);

        int tSum = 0;
        for (String sizeStr : sizesStr) {
            int size = Integer.parseInt(sizeStr);
            tSum += (size + T - 1) / T;
        }

        int pBundle = N / P;
        int pSingle = N % P;

        System.out.println(tSum);
        System.out.println(pBundle + " " + pSingle);
    }
}